app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,71 @@
+class TasksController < ApplicationController
+  def index
+    @all_tasks = Task.all
+    render :index
+  end
+
+  def show
+    @task = Task.find(params[:id])
+    if @task.completed_date.present?
```

Comment:
It might be a good idea to have a `Task#completed?` method to encapsulate the logic of whether or not the task is completed in the model, rather than inspecting that in the controller.